src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -45,6 +46,7 @@ export class PoolListComponent implements OnInit {
     private modalService: BsModalService
   ) {
     this.permission = this.authStorageService.getPermissions().pool;
```

Comment:
To reduce conflicts between both permission:
```suggestion
     this.poolPermission = this.authStorageService.getPermissions().pool;
     this.grafanaPermission = this.authStorageService.getPermissions().grafana;
```

Reply:
To reduce conflicts, I have added the `Permissions` object:
`this.permissions = this.authStorageService.getPermissions();`
and referenced specific permissions with `permissions.grafana.read` and `permissions.pool`